backend/app/synthesize_polygons/views.py

```diff
@@ -34,13 +33,25 @@ def convert_data(data):
     :param data: a dict containing parameters for synthesize_polygon as strings/files
     :return: a dict containing the parameters for synthesize_polygon in correct format
     """
+    float_values = {
+        'noteLength': 'note_length',
+        'noteDelay': 'note_delay',
+        'baseFrequency': 'base_frequency',
+        'floorFrequency': 'floor_frequency',
+        'ceilFrequency': 'ceil_frequency',
+    }
+    bool_values = {
+        'restrictFrequency': 'restrict_frequency',
+        'sidesAsDuration': 'sides_as_duration',
+    }
+
     converted_data = {}
-    if 'noteLength' in data:
-        converted_data['note_length'] = float(data['noteLength'])
-    if 'noteDelay' in data:
-        converted_data['note_delay'] = float(data['noteDelay'])
-    if 'restrictOctave' in data:
-        converted_data['restrict_octave'] = str(data['restrictOctave']).lower() == 'true'
+    for old_name, new_name in float_values.items():
+        if old_name in data:
+            converted_data[new_name] = float(data[old_name])
+    for old_name, new_name in bool_values.items():
+        if old_name in data:
+            converted_data[new_name] = str(data[old_name]).lower() == 'true'
     if 'points' in data:
         converted_data['points'] = [tuple(map(float, p)) for p in data['points']]
```

Comment:
If you find yourself writing a super terse line like this (a map inside a list comprehension with a cast in it) consider: is being this terse worth how hard it's going to be for someone to read this code? I'd probably just write this conversion as a normal for loop and call it a day.